examples/profile-client/src/main/java/com/palantir/example/profile/ProfileStore.java

```diff
@@ -86,24 +86,26 @@ public InputStream getImageForUser(UUID userId) {
             return null;
         }
         UserPhotosStreamStore streamStore = UserPhotosStreamStore.of(txnMgr, tables);
-        return streamStore.loadStream(t, photoId);
+        return streamStore.loadStream(tx, photoId);
     }
 
     public void updateImage(UUID userId, Sha256Hash hash, InputStream imageData) {
         UserProfile userData = getUserData(userId);
-        Preconditions.checkNotNull(userData);
+        Preconditions.checkNotNull(userData, "userData cannot be null");
 
         UserPhotosStreamStore streamStore = UserPhotosStreamStore.of(txnMgr, tables);
         Long oldStreamId = getPhotoStreamId(userId);
         if (oldStreamId != null) {
             // Unmark old stream before we overwrite it.
-            streamStore.unmarkStreamAsUsed(t, oldStreamId, EncodingUtils.encodeUUID(userId));
+            streamStore.unmarkStreamAsUsed(tx, oldStreamId, EncodingUtils.encodeUUID(userId));
         }
 
-        // This will either store a new stream and mark it as used or return an old stream that matches the hash and mark it as used.
-        long streamId = streamStore.getByHashOrStoreStreamAndMarkAsUsed(t, hash, imageData, EncodingUtils.encodeUUID(userId));
+        // This will either store a new stream and mark it as used
+        // or return an old stream that matches the hash and mark it as used.
+        long streamId =
+                streamStore.getByHashOrStoreStreamAndMarkAsUsed(tx, hash, imageData, EncodingUtils.encodeUUID(userId));
```

Comment:
nit: this technically isn't the right place to split the line (after the first `(` and the parameters all on the second line would be ok). Questionable how much we should care for `profile-client`.